### PR TITLE
Tibor

### PR DESCRIPTION
**Pull request by Tibor22**

### CODE REVIEW

File: spec/Bank.spec.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  });
+
+  it("deposit money to card", () => {
+    // setUp
+    bank.deposit("Tibor", 400);
+    // execute
+    const result = bank.searchCard("Tibor").balance;
+
+    //verify
+    expect(result).toEqual(1400);
```
Comment (dogezen) [inline spec/Bank.spec.js:18]: OK, this is a good test. However, how are you storing the transactions? Would be worth to check if the card has the transaction created as well? Your deposit function might update the balance of the card, but forget to save the transaction, so the test at the moment is inconclusive to me

File: spec/Bank.spec.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      transactions: [],
+      balance: 1000,
+      name: "Bob",
+    };
+
+    //verify
+    expect(result1).toEqual(expected1);
+    expect(result2).toEqual(expected2);
+  });
+  it("throw error if deposit is negative or 0", () => {
```
Comment (dogezen) [inline spec/Bank.spec.js:77]: i don't see the test for negative? only for zero.. add an extra expect() statement in this test for a negative amount

File: src/Bank.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+// Dependencies
+
+const Card = require("./Card.js");
+const PrintStatement = require("./printStatement.js");
+
+class Bank {
+  cards = [];
```
Comment (dogezen) [inline src/Bank.js:7]: id' expect to see these proeprties in the constructor because you have a constructor

File: src/Bank.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  deposit(cardName, amount) {
+    const currCard = this.searchCard(cardName);
+    currCard._deposit(amount);
+    this._updateBalance();
+  }
+  withdraw(cardName, amount, category) {
+    const currCard = this.searchCard(cardName);
+    currCard._withdraw(amount, category);
```
Comment (dogezen) [inline src/Bank.js:35]: how do you know if you can withdraw? ie. if i have a balance of 100, can I withdraw 300?

File: src/Card.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+const Deposit = require("./Deposit.js");
+const Withdraw = require("./Withdraw.js");
+
+class Card {
+  transactions = [];
+  balance = 1000;
```
Comment (dogezen) [inline src/Card.js:6]: probably should be initialised to 0?
